be/src/olap/tablet.cpp

```diff
@@ -275,6 +277,23 @@ void Tablet::modify_rowsets(std::vector<RowsetSharedPtr>& to_add,
         same_version = false;
     }
 
+    if (check_delete) {
+        for (auto& rs : to_delete) {
+            auto find_rs = _rs_version_map.find(rs->version());
```

Comment:
maybe need to lock `_meta_lock`

Reply:
It is necessary to get write lock of `_meta_lock` before exec `Tablet::modify_rowsets`. So, we need not lock `_meta_lock ` here.